pkg/config/system_probe.go

```diff
@@ -118,6 +118,7 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.mutex_profile_fraction"), 0)
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.block_profile_rate"), 0)
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.enable_goroutine_stacktraces"), false)
+	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.delta_profiles"), false)
```

Comment:
Shouldn't this default to true ?

Reply:
It was `false` previously (since the config was there), so just maintaining the status quo.

Reply:
I approved the PR, but I think we should enable it by default (maybe in another PR), the performance should be better anyway

Reply:
Or maybe enable it in internal deployment configuration